### DISCUSSION TITLE
MacOS M1 Metal GPU use Tutorial

### DISCUSSION THREAD

**Discussion by Killpit** [mrdbourke/pytorch-deep-learning#336] [Q&A]
I tried to have a look at the internet and I found that no tutorial for MacOS M1 Metal GPU use in PyTorch code. Can someone help me with a reliable tutorial, preferably recent YouTube video etc that I can watch in parts, all help appreciated. I looked at the documentation and I found nothing about it other than CUDA.

**Comment by mrdbourke** [comment]
Hi @Killpit,

See more details on this in the answer here: https://github.com/mrdbourke/pytorch-deep-learning/discussions/337
